dashboard/test/ui/features/i18n.feature

```diff
@@ -0,0 +1,44 @@
+Feature: See Hour of Code and Frozen tutorial in Spanish, Portuguese, and Arabic
+
+Background:
+  Given I am on "http://studio.code.org/reset_session"
+
+Scenario: HoC tutorial in Spanish
+  Given I am on "http://studio.code.org/s/20-hour/stage/2/puzzle/16/lang/es"
+  And I rotate to landscape
+  And I wait for 2 seconds
```

Comment:
What's the purpose of this wait?  It's often more durable to use a more specific wait, like "And I wait to see {some element}".